subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/TaskExecutionAccessCheckers.kt

```diff
@@ -60,6 +60,13 @@ object TaskExecutionAccessCheckers {
     ) : AbstractTaskProjectAccessChecker(broadcaster) {
 
         override fun shouldReportExecutionTimeAccess(task: TaskInternal): Boolean =
-            !configurationTimeBarrier.isAtConfigurationTime
+            !configurationTimeBarrier.isAtConfigurationTime && !temporarilyIgnoreAsWorkaround(task)
+
+        private fun temporarilyIgnoreAsWorkaround(task: TaskInternal): Boolean {
```

Comment:
🔧 Move it to `Workarounds`.

Reply:
Done in https://github.com/gradle/gradle/compare/d23ff36fd2173c72b4ecadbb5030cc23418a10c1..d478204088a6b293046eba396f91f7c2bb3cad6d